install-instructions.php

```diff
@@ -389,6 +389,8 @@
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_SOURCES; ?>" class="btn btn-lg btn-default"><i class="icon-archive"></i> Sources</a>
 		    </div>
 		    <?php } ?>
+                    <h2>Android Beta Client</h2>
```

Comment:
Please take care for consistency by adding the entry like the 'Desktop Clients' entry: **Android Client** [Version]